test_settings.py

```diff
@@ -24,9 +24,28 @@
     },
 }
 
-INSTALLED_APPS = ("platform_plugin_aspects",)
+INSTALLED_APPS = (
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "platform_plugin_aspects",
+)
+
+ROOT_URLCONF = 'platform_plugin_aspects.urls'
+
+SECRET_KEY = 'very-insecure-key'
+
+MIDDLEWARE = [
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+]
 
 EVENT_SINK_CLICKHOUSE_MODEL_CONFIG = {
+    "auth_user": {
+        "module": "django.contrib.auth.models",
+        "model": "User",
+    },
```

Comment:
Adding this model to the settings causes the issues with Celery. For testing, there is no need to configure this model, you can mock the user or permissions. However I think you can disable Celery with CELERY_ALWAYS_EAGER.

Reply:
I tried that, no luck :( Will have another go today.